<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saas_guide_num_key</class>
 <widget class="QDialog" name="saas_guide_num_key">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border:2px solid #ff546b;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>46</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>104</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>46</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>46</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>216</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>104</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>216</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>104</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="public_pushbutton_numkey" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>216</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>public_pushbutton_numkey</class>
   <extends>QPushButton</extends>
   <header>public_pushbutton_numkey.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
